deploy/charts/istio-csr/templates/clusterrole.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if not .Values.ignoreClusterResources }}
 kind: ClusterRole
```

Comment:
For this namespaced approach, you are still going to need to grant these permissions on a per namespace basis right? So keeping this *ClusterRole* would allow users to create namespace-scoped RoleBindings in each namespace that they want enabling for istio-csr use.

Is there a reason to not deploy the ClusterRole? Otherwise this will lead to 1) duplicated resources (everyone needs to create a RoleBinding in every namespace) and 2) uncertainty, as users won't know what permissions to grant in those Roles.

Reply:
If we leave the clusterrole, we can't use the helm charts for more than one deployment, as they will get the same name. The deployments will complain about resources that already exists. The idea is that anything on the clusterlevel is handled outside of this helm chart when running in a multitenant setup. and that the helm charts focus on a single namespace.